src/documents.rs

```diff
@@ -322,7 +322,7 @@ pub struct DocumentDeletionQuery<'a> {
 
     /// Filters to apply.
     ///
-    /// Read the [dedicated guide](https://docs.meilisearch.com/reference/features/filtering.html) to learn the syntax.
+    /// Read the [dedicated guide](https://www.meilisearch.com/docs/learn/fine_tuning_results/filtering#filter-basics) to learn the syntax.
```

Comment:
Not sure if this is the correct replacement, cc: @bidoubiwa

Reply:
it's the correct one :)